src/window_interpreter.cpp

```diff
@@ -153,16 +189,21 @@ void Window_Interpreter::DrawDescriptionLines() {
 
 	contents->TextDraw(rect.x, rect.y, Font::ColorDefault, display_item.desc);
 
-	if (state.wait_movement) {
-		rect = GetItemRect(i++);
-		contents->ClearRect(rect);
-		contents->TextDraw(rect.x, rect.y, Font::ColorCritical, "[WAITING for EV movement!]");
+	rect = GetItemRect(i++);
+	contents->ClearRect(rect);
+
+	if (sub_actions.IsVisible()) {
+		sub_actions.Draw(contents, rect);
 	}
 
 	rect = GetItemRect(i++);
 	contents->ClearRect(rect);
 
-	contents->TextDraw(rect.x, rect.y, Font::ColorDefault, "Stack Size: ");
+	auto str_ex_type = std::string(Game_Interpreter_Shared::kExecutionType.tag(static_cast<int>(stack_display_items[0].type_ex)));
```

Comment:
Just opened the interpreter debug menu with the game from #3409 and it crashed here because stack_display_items is empty.